src/components/APIDoc/ApiDoc.tsx

```diff
@@ -51,5 +52,10 @@ export const ApiDoc: React.FunctionComponent<ApiDocProps> = props => {
             </StackItem>
         )}
         { paths.loading ? <Bullseye><Spinner /></Bullseye> : paths.value }
+        { paths.loading ? <Bullseye><Spinner /></Bullseye>:
+            <StackItem>
+                <SchemaViewer document={ openapi }/>
+            </StackItem>
+        }
```

Comment:
The paths.loading stuff was used to generate the markup in the "background".

```suggestion
        <StackItem>
            <SchemaViewer document={ openapi }/>
        </StackItem>
```

Reply:
We should delete the `const paths` above as well.

Reply:
Ah - hold on - I guess i got something wrong

Reply:
OK - so the schema viewer is the part that shows the types - My initial suggestion still applies

```suggestion
        <StackItem>
            <SchemaViewer document={ openapi }/>
        </StackItem>
```